english/lightsaber/lightsaber.md

```diff
@@ -1,3 +1,10 @@
 You slice through the marshmallow man with your lightsaber. 
 Luckily for you, the heat from it cooks the marshmallows to perfection.
-You now have a tasty snack and a way out of the house. 
\ No newline at end of file
+You now have a tasty snack and a way out of the house. 
+
+But as you hurry to smoosh through the slowly closing hole in the wall, 
+the lightsaber clatters to the floor.
+
+[You leap out nimbly and gaze back at the weapon](lightsaber/wampa.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.